strings/cas_white.txt

```diff
@@ -267,4 +267,29 @@ Wheelchair Ian.
 An actually helpful assistant.
 Mindbreaker toxin.
 Legions of cyborg assholes.
-Homicidal chefs.
\ No newline at end of file
+Homicidal chefs.
+Extended.
+Grammar Nazi gods. 
+Cortical borers.
```

Comment:
guys the rule was added because some guy kept making a ruin full of space nazi's, this is totally acceptable @Cheridan will not mind either way